### DISCUSSION TITLE
Resize Width of MinimizableDialogContainer

### DISCUSSION THREAD

**Discussion by twilio-jyoung** [twilio-labs/paste#2762] [Q&A] [answered]
There doesn't seems to be an editable width property on the MinimizableDialogContainer.  

It's hardcoded in index.es to 

`width: "size40",`

**Comment by gloriliale** [comment]
Hi @twilio-jyoung! Are you coming across a use case that requires the component to be a different width? Feel free to provide more context so that we can understand where your concern lies!

**Reply by twilio-jyoung** [reply]
the flex SE team is building an email component (like gmail), but the width is too narrow and unmodifiable on MinimizableDialogContainer to render what we want to render inside ([nylas composer](https://developer.nylas.com/docs/user-experience/components/composer-component/)) nicely.

![Screen Shot 2022-10-25 at 4 48 30 PM](https://user-images.githubusercontent.com/44909531/197888575-4a77eb6a-431d-455e-a616-88954b008a8a.png)

**Reply by shleewhite** [reply]
@twilio-jyoung Is this a flex plugin? If so, you can use the CustomizationProvider to change the width.

**Reply by loreina** [reply]
@twilio-jyoung Hey! Just following up here. Was Lee's answer able to help solve the issue or how can we help resolve this? Thanks!

**Comment by loreina** [comment, marked as answer]
👋 @twilio-jyoung!

If this is a Flex plugin, you can use the [Customization Provider](https://paste.twilio.design/customization/customization-provider) to change the width. Marking this as answered though feel free to comment to re-open this discussion, thanks!
